commons/monitor/questionnaire/questionnaire_application_usage_log.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "namespace": "org.radarcns.monitor.questionnaire",
+  "type": "record",
+  "name": "QuestionnaireApplicationUsageLog",
+  "doc": "Schema of reporting events when opening the app, clicking notifications, and starting and finish questionnaires.",
```

Comment:
Possible alternative:
`Event for user interaction with the questionnaire application. This describes significant user interactions, such as response to notifications and starting or completing questionnaires.`